lib/codit/outbreak.py

```diff
@@ -93,9 +122,29 @@ def record_step(self, o):
                 # len([t for t in all_completed_tests if t.positive]) / N / o.time_increment,
                 # tot_haz/pot_haz,
                 ]
-        if o.step_num % (50 * o.society.episodes_per_day) == 1 or (o.step_num == o.n_periods):
+        # Increase the update frequency to have more heatmap images to show in video
+        if o.step_num % (7 * o.society.episodes_per_day) == 1 or (o.step_num == o.n_periods):
             logging.info(f"Day {int(step[0])}, prop infected is {step[1]:2.2f}, "
                          f"prop infectious is {step[2]:2.4f}")
+            # Generate heatmaps for infectious people along the time, feed heatmaps into camera
+            if o.pop.count_infectious() > 0:
```

Comment:
The below becomes a method of the new Object ...

Reply:
Yes, will do.